CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.0.1, 13 May 2020
```

Comment:
Minor but I think this should be `v1.1.0`. Could you also change the version in `lib/onfido/version.rb`?

```suggestion
## v1.1.0, 13 May 2020
```

Reply:
Thanks! I've just did that and I've also updated one other instance on README.md